angular-workspace/projects/ni/nimble-angular/.eslintrc.js

```diff
@@ -4,7 +4,8 @@ module.exports = {
     extends: '../../../.eslintrc.js',
```

Comment:
- We can use the tag versions of urls to make it easier to see which version of angular these were forked from, ie https://github.com/angular/angular/blob/12.1.5/packages/forms/src/directives/control_value_accessor.ts
- We should make sure these files are being pulled from the same version of angular. Based on the radio control value accessor url it looks like it was copied from the v14 change not v12 (I did not check to see if there are significant changes to be aware of). It's possible we could just stay on the v14 changes if you inspect the differences from our v12 to the v14
- We should add a README to the thirdparty folder that describes where the directives folder was sourced from and gives update instructions
- We should update the nimble-angular CONTRIBUTING to add a section on performing Angular updates to point to the thirdparty folder

Reply:
Did all the things. After examining the diffs, I determined that it is fine to just use the 14.2.6 (last version before 15.0.0) version of the files. One was identical, and the other had just a couple new `override` keywords on existing member functions.